### PR TITLE
why can't I construct jacobian this way?

### PR DESCRIPTION
**Pull request by wdednam**
This change is [Reviewable](https://reviewable.io/reviews/highperformancecoder/minsky/245)

### CODE REVIEW

File: model/minsky.cc [hunk 1]
```diff
@@ -1078,17 +1078,27 @@ namespace minsky
         ds[j]=1;
         for (size_t i=0; i<equations.size(); ++i)
           equations[i]->deriv(&df[0], df.size(), &ds[0], sv, &flow[0]);
-        vector<double> d(stockVars.size());
-        evalGodley.eval(&d[0], &df[0]);
+        double d[stockVars.size()];
```
Comment (highperformancecoder) [inline model/minsky.cc:1081]: This is not standard C++.

**Comment by highperformancecoder** [comment]
You might need to check that the d1 & d2 definitions are correct for tensor operations. They may not be.

File: model/minsky.cc (hunk 1 above)
Comment (wdednam) [reply model/minsky.cc:1081]: Whoops. i meant to change it back and forgot...

**Comment by wdednam** [comment]
> You might need to check that the d1 & d2 definitions are correct for tensor operations. They may not be.

Thank you, I will. At first I thought it was just the size of the Jacobian that may be the problem, but it is not as simple as that.

**Comment by wdednam** [comment]
Crap, I thought I had a separate branch for this. Going to revert now and submit a different PR. Sorry...

**Comment by highperformancecoder** [comment]
What's happening with this PR?

**Comment by wdednam** [comment]
> What's happening with this PR?

Unfortunately. not much. I spent more time on this after your suggestion but did not make much progress. So, you can close it if you want to.

**Pull request closed without merging**